pandas/core/arrays/categorical.py

```diff
@@ -316,50 +316,19 @@ class Categorical(ExtensionArray, PandasObject):
     def __init__(self, values, categories=None, ordered=None, dtype=None,
                  fastpath=False):
 
-        # Ways of specifying the dtype (prioritized ordered)
```

Comment:
it might be worth moving these pseudo instructions to ``CategoricalDtype._from_values_or_dtype``

Reply:
Something similar is a good idea, but this specific list was for all of the construction in the  __init__, so not always correct for ``CategoricalDtype._from_values_or_dtype``. I'll adjust the list.